frameworks/cassandra/RELEASE_NOTES.md

```diff
@@ -0,0 +1,50 @@
+### Package Upgrade in Strict Mode
```

Comment:
Would you mind putting the following comment tags around this text? In the future I'm going to try to create just one document rather than duplicating info.

<!-- Begin text duplicated for all services -->

<!-- End text duplicated for all services -->